src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -19,14 +19,14 @@ public Movie add(Movie movie) {
         try {
             session = HibernateUtil.getSessionFactory().openSession();
             transaction = session.beginTransaction();
-            session.save(movie);
+            session.persist(movie);
```

Comment:
why do you have to change the method save()? it is working, even if idea write to you we don't have this task ;)